extensions/resteasy-reactive/rest-jackson/deployment/src/main/java/io/quarkus/resteasy/reactive/jackson/deployment/processor/JacksonSerializerFactory.java

```diff
@@ -130,10 +139,39 @@ public Collection<String> create(Collection<ClassInfo> classInfos) {
             create(toBeGenerated.removeFirst()).ifPresent(createdClasses::add);
         }
 
+        createFieldNamesClass();
+
         return createdClasses;
     }
 
-    public Optional<String> create(ClassInfo classInfo) {
+    public void createFieldNamesClass() {
```

Comment:
Can we have some Javadoc describing what this generates and ideally an example of such a generated class too?

Thanks

Reply:
Done.